awswrangler/timestream.py

```diff
@@ -10,7 +10,7 @@
 import pandas as pd
```

Comment:
Should we extend our tests to multiple measure cols?

Reply:
Yep, good point. I only fixed the existing code in the PR